test/augmentation/test_backward.py

```diff
@@ -212,7 +209,7 @@ def test_param(
             elif resample == 'nearest' or torch.all(aug._param_generator.translate._grad == 0.0):
                 # grid_sample will return grad = 0 for resample nearest
                 # https://discuss.pytorch.org/t/autograd-issue-with-f-grid-sample/76894
-                assert (translate.to(device=device, dtype=dtype) - aug._param_generator.translate.data).sum() == 0
+                assert_close(translate, aug._param_generator.translate)
             else:
                 assert (translate.to(device=device, dtype=dtype) - aug._param_generator.translate.data).sum() != 0
```

Comment:
@shijianjian i don't understand this test. for fp32 this difference is `eps` but e.g in fp16 it's absolute zero. I believe we should revisit all this tests